### PR TITLE
Stream update 2.4.9

### PR DESCRIPTION
**Pull request by Silv3rcircl3**

### CODE REVIEW

File: src/core/Akka.Streams.TestKit.Tests/TestPublisherSubscriberSpec.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ namespace Akka.Streams.TestKit.Tests
 {
     public class TestPublisherSubscriberSpec : AkkaSpec
     {
-        protected readonly ActorMaterializer Materializer;
+        private ActorMaterializer Materializer { get; }
```
Comment (alexvaluyskiy) [inline src/core/Akka.Streams.TestKit.Tests/TestPublisherSubscriberSpec.cs]: why you need private property?

File: src/core/Akka.Streams.Tests/Implementation/Fusing/InterpreterStressSpec.cs [hunk 2]
```diff
@@ -30,7 +30,7 @@ public class InterpreterStressSpec : GraphInterpreterSpecKit
         private readonly ITestOutputHelper _helper;
 
         // GraphStages can be reused
-        private static readonly Select<int,int> Select = new Select<int, int>(x=>x+1, Deciders.StoppingDecider);
+        private static readonly Select<int,int> Select = new Select<int, int>(x=>x+1);
```
Comment (alexvaluyskiy) [inline src/core/Akka.Streams.Tests/Implementation/Fusing/InterpreterStressSpec.cs]: Spaces in lambda

File: src/core/Akka.Streams.TestKit.Tests/TestPublisherSubscriberSpec.cs (hunk 1 above)
Comment (Silv3rcircl3) [inline src/core/Akka.Streams.TestKit.Tests/TestPublisherSubscriberSpec.cs]: For running [this](https://github.com/akkadotnet/akka.net/pull/2335/files/e92e271ee14fa32ed9d4ef6aca0335b2fabae351#diff-4549bb036193f22a3c487a399f396d02R34) stream.

**Comment by Silv3rcircl3** [comment]
@alexvaluyskiy addressed your remarks

**Pull request merged**